apis/rollout/well_known_labels.go

```diff
@@ -15,8 +15,10 @@
 package rollout
 
 const (
-	LabelControl     = "rollout.kusionstack.io/control"
-	LabelCreatedBy   = "rollout.kusionstack.io/created-by"
-	LabelWorkload    = "rollout.kusionstack.io/workload"
-	LabelGeneratedBy = "rollout.kusionstack.io/generated-by"
+	LabelControl        = "rollout.kusionstack.io/control"
+	LabelCreatedBy      = "rollout.kusionstack.io/created-by"
+	LabelWorkload       = "rollout.kusionstack.io/workload"
+	LabelGeneratedBy    = "rollout.kusionstack.io/generated-by"
+	LabelRolloutID      = "rollout.kusionstack.io/rollout-id"
+	LabelRolloutBatchID = "rollout.kusionstack.io/rollout-batch-id"
```

Comment:
add a new annotation `rollout.kusionstack.io/process-info` instead of LabelRolloutID and LabelRolloutBatchID

the process-info struct is defined in apis/rollout/v1alpha1: 

```
// ProgressingInfo is the rollout progressing info
type ProgressingInfo struct {
	RolloutName    string                 `json:"rollout,omitempty"`
	RolloutRunName string                 `json:"rolloutRun,omitempty"`
	Canary         *CanaryProgressingInfo `json:"canary,omitempty"`
	Batch          *BatchProgressingInfo  `json:"batch,omitempty"`
}

```

Reply:
ok